test/functional/core/spellfile_spec.lua

```diff
@@ -11,15 +11,18 @@ local write_file = t.write_file
 local mkdir = t.mkdir
 
 local testdir = 'Xtest-functional-spell-spellfile.d'
+local old_xdg_data = os.getenv('XDG_DATA_HOME')
 
 describe('spellfile', function()
   before_each(function()
     clear()
+    fn.setenv('XDG_DATA_HOME', old_xdg_data)
```

Comment:
instead of this, just call clear() with `env`:

    clear{ env = { ... } }

then you don't need to save/restore `old_xdg_data`.

Reply:
Perfect, thank you!